package.json

```diff
@@ -33,6 +33,8 @@
   "dependencies": {
     "babel-core": "6.9.1",
     "babel-plugin-syntax-jsx": "6.8.0",
-    "babel-plugin-transform-flow-strip-types": "6.8.0"
+    "babel-plugin-transform-flow-strip-types": "6.8.0",
+    "fs": "0.0.1-security",
+    "path": "^0.12.7"
```

Comment:
This is interesting. What is the advantage of using this  package as opposed to the native "path" module?